src/zenml/pipelines/base_pipeline.py

```diff
@@ -164,6 +170,15 @@ def enable_cache(self) -> bool:
         """
         return self.configuration.enable_cache
 
+    @property
```

Comment:
I would remove this property tbh, it's already accessible via `pipeline.configuration.enable_artifact_metadata`. The only reason the other shortcuts exist is because of backwards compatability reasons

Reply:
Ah I see. Sure, let's get rid of it then

Reply:
discussion: just leave them for now since we haven't even deprecated them